### PR TITLE
docs: Update KPR limitations wrt IPsec

### PR DESCRIPTION
**Pull request by raymonddejong**
KPR is supported with Wireguard, but not with IPSec. This update of the docs reflects that.

Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Thanks for contributing!

Update of the KPR limitations page as KPR is supported with Wireguard but not with IPSec.

```release-note
<!-- Enter the release note text here if needed or remove this section! -->
```

### CODE REVIEW

**Review by brb — COMMENTED** [review]
LGTM, thanks! I will ACK once you sign off the commit.

File: Documentation/network/kubernetes/kubeproxy-free.rst
Comment (pchaigno) [inline Documentation/network/kubernetes/kubeproxy-free.rst]: The second commit needs to be squashed in the first. You can do that with:
```
git reset HEAD^
git add Documentation/network/kubernetes/kubeproxy-free.rst
git commit --amend
git push -f
```

File: Documentation/network/kubernetes/kubeproxy-free.rst
Comment (pchaigno) [inline Documentation/network/kubernetes/kubeproxy-free.rst:1413]: Could you update the title of the first commit (and of the PR) to something more descriptive? Maybe `docs: Update KPR limitations wrt WireGuard`?

**Review by pchaigno — CHANGES_REQUESTED** [review]
Thanks for the fix!

**Review by brb — APPROVED** [review]

**Review by pchaigno — APPROVED** [review]

**Comment by pchaigno** [comment]
The Documentation workflow passed. There are no changes to the docs structure. Merging.

**Pull request merged**
